subprojects/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r26/TestLauncherCrossVersionSpec.groovy

```diff
@@ -495,24 +493,8 @@ class TestLauncherCrossVersionSpec extends TestLauncherSpec {
     }
 
     def changeTestSource() {
-        // adding two more test methods
-        file("src/test/java/example/MyTest.java").text = """
-            package example;
-            public class MyTest {
-                @org.junit.Test public void foo() throws Exception {
-                     org.junit.Assert.assertEquals(1, 1);
-                }
-                @org.junit.Test public void foo2() throws Exception {
-                     org.junit.Assert.assertEquals(1, 1);
-                }
-                @org.junit.Test public void foo3() throws Exception {
-                     org.junit.Assert.assertEquals(1, 1);
-                }
-                @org.junit.Test public void foo4() throws Exception {
-                     org.junit.Assert.assertEquals(1, 1);
-                }
-            }
-        """
+        // Removes MyTest.class to trigger a new build because it's input of test task.
```

Comment:
Can you add a comment here why we don't change the source file?

Reply:
Isn't commit message enough?

Reply:
I think the commit message is pretty hard to find when you look at the code. I'd prefer having another comment in the code itself.